### DISCUSSION TITLE
FTP Connection Export

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#289] [Suggestions]
Is it possible to have all FTP Connections in one file to import in our FTP App like FileZilla for example?

**Comment by furio85** [comment]
I agree, would be nice.  I use file-zilla at times.  But I have also found that all you need to do is create the links once.  If you save the file-zilla save and just back it up.  You can send it to other admins if you wish.  Only takes 30 sec to add a server anyway.

**Comment by mvaessen** [comment]
From a security perspective I do think this would result in a higher risk of people accidentally leaking their FTP passwords. So because of that I'm going to reject.
